CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ This is the log of notable changes to EAS CLI and related packages.
 ### 🧹 Chores
 
 - Limit project file upload size to 2GB. ([#1928](https://github.com/expo/eas-cli/pull/1928) by [@khamilowicz](https://github.com/khamilowicz))
+- Skip noUnusedLocals and noUnusedParameters during local development. ([#1933](https://github.com/expo/eas-cli/pull/1933) by [@khamilowicz](https://github.com/khamilowicz))
```

Comment:
I think that there's no need to add this to the changelog. Changelog should inform users about what changed in the given version. This is just for local dev.